app/backend/src/couchers/servicers/api.py

```diff
@@ -33,7 +33,7 @@
 )
 from couchers.resources import language_is_allowed, region_is_allowed
 from couchers.sql import couchers_select as select
-from couchers.tasks import send_friend_request_email, send_report_email, send_friend_request_accepted_email
+from couchers.tasks import send_friend_request_accepted_email, send_friend_request_email, send_report_email
```

Comment:
Revert unrelated change

Reply:
This is just from isort and black, develop hadn't been formatted.

I can move it into another PR if you want though?

Reply:
Yes with a link to the offending PR that was merged despite not being formatted

Reply:
I'll make a separate PR